pocs/utils/data.py

```diff
@@ -12,18 +12,22 @@ def download_all_files(data_folder=None):
     if data_folder is None:
         data_folder = "{}/astrometry/data".format(os.getenv('PANDIR'))
 
-    for i in range(4210, 4219):
-        fn = 'index-{}.fits'.format(i)
-        dest = "{}/{}".format(data_folder, fn)
-
+    def download_one_file(fn):
+        dest = "{}/{}".format(data_folder, os.path.basename(fn))
         if not os.path.exists(dest):
-            url = "http://data.astrometry.net/4200/{}".format(fn)
+            url = "http://data.astrometry.net/{}".format(fn)
             df = data.download_file(url)
             try:
                 shutil.move(df, dest)
             except OSError as e:
                 print("Problem saving. (Maybe permissions?): {}".format(e))
 
+    for i in range(4112, 4119):
```

Comment:
Just reproduced the test failure on my local and it's because we are missing 4110 and 4111 index files. The fits files we use for testing have a small field-of-view, which is what those index files will search. 10 and 11 are fairly large (24MB and 10MB) so this is a bit annoying as we only really need them for the tests.

I also removed the 42xx from my local and all tests still pass. The 41xx are wide-field so should be all that we need although I suppose it doesn't hurt to have all of them.

Reply:
OK. Can we replace the file used for testing to a wide field also?